recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/xenpmd.te

```diff
@@ -50,7 +50,7 @@ dbus_connect_system_bus(xenpmd_t)
 dbus_send_system_bus(xenpmd_t)
 uid_dbus_chat(xenpmd_t)
 surfman_dbus_chat(xenpmd_t)
-rpcproxy_websockets_dbus_chat(xenpmd_t)
+rpcbroker_dbus_chat(xenpmd_t)
```

Comment:
Why does this change from the websockets variant to the plain one?  Your SELinux policy includes rpcbroker_websockets_dbus_chat.

Reply:
Yeah this is confusing... Do you know what the best practice is for allowing 2 domains to "dbus_chat" with each other? Should they both "call" the interface of the other one? Or maybe only the domain that will be the caller should "call" the necessary interface?

Reply:
Got it thanks! I will centralize everything in the rpc-broker policy module.